test_pull_requests

```diff
@@ -2089,7 +2089,8 @@ popd
       # First, get the next jenkins build number
       next_build_num = get_next_build(branch, settings)
 
-      pull_id_params = "&BRANCH=#{branch}"
+      branch_param = settings["branch_param"] || "BRANCH"
```

Comment:
I wouldn't want to have to set all the values because I have 1 job which is different.